Include/arm_nnsupportfunctions.h

```diff
@@ -370,6 +370,35 @@ arm_cmsis_nn_status arm_nn_mat_mult_nt_t_s8(const int8_t *lhs,
                                             const int32_t activation_max,
                                             const int32_t lhs_cols_offset);
 
+/**
```

Comment:
Please update the file version and date..

Reply:
Done